### DISCUSSION TITLE
Can a `>>=` ligature be enabled?

### DISCUSSION THREAD

**Discussion by schuelermine** [tonsky/FiraCode#1402] [Q&A] [answered]
I assume `>>=` isn’t a flipped version of `=<<` because of the JS operator `>>=`, as that’s also the reasoning given by the Cascadia Code developers, but can it be enabled?

**Comment by tonsky** [comment, marked as answer]
ss09 should do it

![](https://github.com/tonsky/FiraCode/raw/master/extras/ligature_variants.png)

**Comment by schuelermine** [comment]
Thanks! Can I enable it globally or does it need to be enabled per application?

**Reply by tonsky** [reply]
Per app. See https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets
